### DISCUSSION TITLE
Run in multiple window

### DISCUSSION THREAD

**Discussion by Uno-count** [seleniumbase/SeleniumBase#2655] [Q&A] [answered]
I have a test_stress.py that contains a single function def test_stress(self):

Now I want to run it in multiple windows simultaneously. Is there a way to do that?

Thank you very much in advance.

**Comment by mdmintz** [comment, marked as answer]
See [SeleniumBase/examples/test_repeat_tests.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_repeat_tests.py)
(It uses `pytest` multithreading. Eg: `pytest -n6`)
